extensions/rich_text_components/Link/Link.js

```diff
@@ -27,14 +27,8 @@ oppia.directive('oppiaNoninteractiveLink', [
       templateUrl: 'richTextComponent/Link',
       controller: ['$scope', '$attrs', 'explorationContextService',
         function($scope, $attrs, explorationContextService) {
-          if (!$attrs.openLinkInSameWindowWithValue) {
-            // This is done for backward-compatibility.
-            $scope.target = '_blank';
-          } else {
-            $scope.target = (
-              oppiaHtmlEscaper.escapedJsonToObj(
-                $attrs.openLinkInSameWindowWithValue) ? '_top' : '_blank');
-          }
+          // Always open link in new tab/window
+          $scope.target = '_blank';
```

Comment:
Since we know `target` is `"_blank"`, just write `target="_blank"` in the html template for this directive, (see `extensions/rich_text_components/Link/Link.html`). Then you can get rid of the `$scope.target` variable.